google-cloud-tools-plugin/resources/META-INF/google-cloud-maven.xml

```diff
@@ -18,4 +18,5 @@
 <idea-plugin xmlns:xi="http://www.w3.org/2001/XInclude">
     <xi:include href="/META-INF/app-engine-java-maven.xml" xpointer="xpointer(/idea-plugin/*)"/>
     <xi:include href="/META-INF/google-cloud-apis.xml" xpointer="xpointer(/idea-plugin/*)"/>
```

Comment:
does this one still belong here?

Reply:
still does to make it work, once we have all Maven dependencies gone into `maven-apis` module (for now it's just an empty placeholder sub-module), we can safely remove this line.